podcasts/Whats New/WhatsNewView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct WhatsNewView: View {
+    @EnvironmentObject var theme: Theme
+
+    let announcement: WhatsNewAnnouncement
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ZStack(alignment: .top) {
+                Rectangle()
+                    .frame(height: 195)
+
+                HStack {
+                    Spacer()
+                    Button {
+                        NavigationManager.sharedManager.dismissPresentedViewController()
+                    } label: {
+                        ZStack {
+                            Image("close")
+                                .foregroundColor(.white)
+                        }
+                        .frame(width: 44, height: 44)
+                    }
+                }
+            }
+            Text(announcement.title)
+                .font(style: .title3, weight: .bold)
+                .padding(.horizontal)
+                .padding(.top)
+                .foregroundColor(theme.primaryText01)
+            Text(announcement.message)
+                .font(style: .subheadline)
+                .foregroundColor(theme.secondaryText02)
+                .padding(.horizontal)
+                .padding(.bottom)
+            Button("Enable it") {}
+                .buttonStyle(RoundedDarkButton(theme: theme))
+                .padding(.horizontal)
+                .padding(.bottom)
+        }
+        .frame(width: 340)
+        .background(.white)
```

Comment:
In some themes, the text is completely hidden.

<img width="530" alt="Screenshot 2023-07-05 at 4 11 59 PM" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/a3c23fba-5918-4477-b7ca-82b88480622d">


```suggestion
        .background(theme.primaryUi01)
```